pkgs/applications/networking/instant-messengers/rambox-pro/default.nix

```diff
@@ -0,0 +1,46 @@
+{ autoPatchelfHook, electron, fetchurl, makeDesktopItem, makeWrapper, nodePackages, nss, stdenv, xdg_utils, xorg }:
+
+stdenv.mkDerivation rec {
+  name = "rambox-pro-${version}";
+  version = "1.1.2";
+
+  dontBuild = true;
+  dontStrip = true;
+
+  buildInputs = [ nss xorg.libxkbfile ];
+  nativeBuildInputs = [ autoPatchelfHook makeWrapper nodePackages.asar ];
+
+  src = fetchurl {
```

Comment:
Is it possible to use fetchFromGithub?

Reply:
I'm completely open to feedback on this one but from what I understand fetchFromGithub is aimed at fetching a commit from the repository but sadly in this case there are no commits after the initial README.md because the application is "nonfree" so no source code is provided.

The only place to grab the releases is under "Releases" on GitHub.

Again though - this may be possible and I just don't know about it.

Reply:
Ah i didn't realise, I'm not really sure on this one

Reply:
It's ok to use `fetchurl` here **only** because this is a _release_ linked as an asset to a tag on the repo, and not a source archive of the repo content as generated by GitHub